torchdynamo/resume_execution.py

```diff
@@ -49,6 +45,24 @@ def __call__(self, code_options, cleanup):
             create_instruction("POP_TOP"),
         ]
 
+    # For Python 3.7 compatibility
+    def _create_compatible_begin_finally(self, code_options):
+        if sys.version_info < (3, 8):
+            co_consts = code_options["co_consts"]
```

Comment:
Could we refactor this to share code with _create_load_const?  

You could modify `_create_load_const` to have a `@staticmethod` helper function.

Reply:
Good point! Let me do it!